packages/i18nify-go/country.go

```diff
@@ -42,9 +43,29 @@ func (c Country) GetCountryCurrency() []currency.CurrencyInformation {
 	return curInfoList
 }
 
+// GetStatesByZipCode retrieves the states with zipcode for the country.
+func (c *Country) GetStatesByZipCode(pinCode string) []subdivisions.State {
```

Comment:
A state will have list of cities too right?

Reply:
Yes

Reply:
@rzp-Piyush Then do we need getcities by pincode?

Reply:
the state will contain all the cities, If the functionality requires the exact city to be fetched then yes we need it.

Reply:
we can just give the state object that will contain only the cities for which the postal code is matched. Instead of keeping two functionalities this can be merged into one

Reply:
Yes, please merge them

Reply:
merged